homeassistant/components/netgear_lte/__init__.py

```diff
@@ -67,7 +67,15 @@
     {
         vol.Optional(
             CONF_MONITORED_CONDITIONS, default=sensor_types.DEFAULT_SENSORS
```

Comment:
I don't remember if we've already talked about this, but is this option compatible with adr 3?

https://github.com/home-assistant/architecture/blob/master/adr/0003-monitor-condition-and-data-selectors.md

Reply:
We didn't, it predates ADR 3.

I think ADR 3 is quite vague, maybe because it was just suddenly created without addressing the concerns in the issue. So I am not sure if this is compatible.

I would hope that _not_ automatically adding 100+ (I currently have 412) sensors falls within "if it makes sense".

Reply:
Sounds reasonable! :+1:

Reply:
I started writing a comment explaining how it all ties together, then realized it should probably get more visibility, so opened architecture issue https://github.com/home-assistant/architecture/issues/276

Reply:
So where does that put this PR? Should I close it until that stuff actually exists?